src/components/common/store/searchReducer.tsx

```diff
@@ -116,6 +117,13 @@ const searchResult = async (
     p.sortby = param.sortby;
   }
 
+  // Track analytics only for list searches to avoid duplicate events
```

Comment:
Please move the if check into the trackSearchUrlParameter as this is something track wants to know but not others